src/bsd.c

```diff
@@ -1127,7 +1127,8 @@ handle_curl_msg(CURLMsg *msg)
                                 NULL, NULL);
     } else {
       notify_format(resp->thing, "Request failed: %s",
-                    curl_easy_strerror(msg->data.result));
+                    resp->too_big ? "Too much data received."
```

Comment:
Hm, do we want to do this, or truncate instead?

Reply:
The problem with truncation (which is the current behavior that I'm not really thrilled about) is... well... what are you going to do with half of a JSON object?

Reply:
Truncating JSON is bad, but I'm thinking, e.g: Fetch a wikipedia article, and grab the first paragraph. Things like that may be doable with truncation. And encouraging `@assert strmatch(%q<status>, 200 *) ; @assert isjson(%0)` might be a good idea since there's other ways it can fail (gateway redirect, etc) than just exceeding `BUFFER_LEN`.